webcompat/helpers.py

```diff
@@ -187,19 +187,24 @@ def get_os(user_agent_string=None):
     return "Unknown"
 
 
-def get_response_headers(response):
+def get_response_headers(response, mime_type=JSON_MIME):
     """Return a dictionary of headers based on a passed in Response object.
 
     This allows us to proxy response headers from GitHub to our own responses.
     """
-    headers = {'etag': response.headers.get('etag'),
-               'cache-control': response.headers.get('cache-control'),
-               'content-type': JSON_MIME}
-
-    if response.headers.get('link'):
-        headers['link'] = rewrite_and_sanitize_link(
-            response.headers.get('link'))
-    return headers
+    # handle the case where we get the data directly
+    headers = {}
+    if isinstance(response, requests.models.Response):
+        headers = response.headers
+    # or the case where we proxy an already fetched reponse
+    if isinstance(response, tuple):
+        headers = response[2]
+    new_headers = {'etag': headers.get('etag'),
+                   'cache-control': headers.get('cache-control'),
+                   'content-type': mime_type}
```

Comment:
if by chance the two `if` fails. We end up with  `headers.get('etag')` on `{}`   aka `'etag': None`. 
Maybe it's working or does it 💣

Maybe ok? 😆 

```python
>>> Response('foo', 200, {'etag': None}).headers
Headers([('etag', 'None'), ('Content-Type', 'text/html; charset=utf-8'), ('Content-Length', '3')])
```

Reply:
:))~ yeah, I guess the worst case scenario here is there's no cache-hit.

(I hope)